libs/Microsoft.MixedReality.WebRTC.Native/test/audio_transceiver_tests.cpp

```diff
@@ -0,0 +1,241 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
All the tests in this file are replicated word by word in video_transceiver_test.cpp. Should we write them just once, maybe parameterize them on the media kind and run twice? (though I suspect that running them once would be good enough).

Reply:
Fixed. This makes something like this once parameterized by SDP semantic and by media type:

![image](https://user-images.githubusercontent.com/49064175/75817296-b31d4300-5d8e-11ea-973a-0feb62c7bc2b.png)